marm-mcp-server/tests/test_notebook_service.py

```
@@ -0,0 +1,127 @@
+async def test_dispatch_use_activates_existing_entry(notebook_svc):
+    dispatch, memory = notebook_svc
+    await dispatch(action="add", name="rule_c", data="cite sources")
+    result = await dispatch(action="use", names="rule_c")
+    assert result["status"] == "success"
+    assert "rule_c" in result["activated_entries"]
+    assert memory.active_notebook_entries[0]["name"] == "rule_c"
+
+
+@pytest.mark.asyncio
+async def test_dispatch_status_reflects_active_entries(notebook_svc):
+    dispatch, memory = notebook_svc
```

Ruff (0.15.14)

[warning] Unpacked variable `memory` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)